jsapp/js/endpoints/organization.mocks.ts

```diff
@@ -0,0 +1,29 @@
+import { http, HttpResponse } from 'msw'
+import type { OrganizationResponse } from '#/api/models/organizationResponse'
+import { getOrganizationsRetrieveUrl } from '#/api/react-query/organizations'
+
+/**
+ * Mock API for session endpoint. Use it in Storybook tests in `parameters.msw.handlers[]`.
+ */
+const organizationMock = (organizationId: string) =>
+  http.get<never, never, OrganizationResponse>(getOrganizationsRetrieveUrl(organizationId), () =>
+    HttpResponse.json(organizationReponse(organizationId)),
+  )
+export default organizationMock
+
+const organizationReponse = (organizationId: string): OrganizationResponse => ({
+  id: organizationId,
+  url: 'http://kf.kobo.local/api/v2/organizations/orgaziUixYLNArojSY6MYnxA/',
```

Comment:
```suggestion
  url: `http://kf.kobo.local/api/v2/organizations/${organizationId}/`,
```
Not sure if this would be problematic in any way in current code, but wouldn't harm to use id here too